ubuntudesign/developer.ubuntu.com
# Update to Vanilla v1.8

## Done
- Updated version of site to v1.8
- Removed margins from p.content and p.tutorial cards
- Updated `h3` to use correct markup

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8015/](http://0.0.0.0:8015/)
4. Compare with [live](https://developer.ubuntu.com/) and ensure that the site looks the same (or better)
